static/lib/plugins.json

```diff
@@ -1,4 +1,15 @@
 [
+  {
+    "name": "Design Patterns",
```

Comment:
I'd rename it to "Boilerplate" instead. Please also respect the alphabetical order of the array.